input/tests/measure/UrinarySymptomScoreChangeAfterBenignProstaticHyperplasiaFHIR/623a7f40-c265-4e42-a5c5-6f54e20b19df/CMS771FHIR-v1.3.000-DENEXFail-UrinaryRetentionDxUnconfirmed.json

```diff
@@ -0,0 +1,363 @@
+    "request": {
+      "method": "POST",
+      "url": "Observation/55509234"
+    }
+  }, {
+    "fullUrl": "https://madie.cms.gov/MeasureReport/56480444-7edf-4c77-8c13-66519089395c",
+    "resource": {
+      "resourceType": "MeasureReport",
+      "id": "56480444-7edf-4c77-8c13-66519089395c",
+      "meta": {
+        "profile": [ "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/test-case-cqfm" ]
+      },
+      "contained": [ {
+        "resourceType": "Parameters",
+        "id": "12ebb106-4d06-4eb6-9492-20a63930ef08-parameters",
+        "parameter": [ {
+          "name": "subject",
+          "valueString": "623a7f40-c265-4e42-a5c5-6f54e20b19df"
+        } ]
+      } ],
+      "extension": [ {
+        "url": "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-inputParameters",
+        "valueReference": {
+          "reference": "#12ebb106-4d06-4eb6-9492-20a63930ef08-parameters"
+        }
+      }, {
+        "url": "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-testCaseDescription",
+        "valueMarkdown": "Patient meeting initial population and denominator criteria fails denominator exclusion despite having Urinary Retention Diagnosis because it is of unconfirmed status"
+      } ],
+      "modifierExtension": [ {
+        "url": "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-isTestCase",
+        "valueBoolean": true
+      } ],
+      "status": "complete",
+      "type": "individual",
+      "measure": "https://madie.cms.gov/Measure/UrinarySymptomScoreChangeAfterBenignProstaticHyperplasiaFHIR",
+      "period": {
+        "start": "2025-01-01",
+        "end": "2025-12-31"
+      },
+      "group": [ {
+        "population": [ {
+          "code": {
+            "coding": [ {
+              "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+              "code": "initial-population",
+              "display": "Initial Population"
+            } ]
+          },
+          "count": 1
+        }, {
+          "code": {
+            "coding": [ {
+              "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+              "code": "denominator",
+              "display": "Denominator"
+            } ]
+          },
+          "count": 1
+        }, {
+          "code": {
+            "coding": [ {
+              "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+              "code": "denominator-exclusion",
+              "display": "Denominator Exclusion"
+            } ]
+          },
+          "count": 0
+        }, {
+          "code": {
+            "coding": [ {
+              "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+              "code": "numerator",
+              "display": "Numerator"
+            } ]
+          },
+          "count": 0
+        } ]
+      } ],
+      "evaluatedResource": [ {
+        "reference": "https://madie.cms.gov/Patient/623a7f40-c265-4e42-a5c5-6f54e20b19df"
```

Comment:
External references do not work you have to strip off the https://madie.cms.gov/ from these